api/lib/infrastructure/repositories/certification-cpf-country-repository.js

```diff
@@ -0,0 +1,14 @@
+const { CertificationCpfCountry } = require('../../domain/models/CertificationCpfCountry');
+const { knex } = require('../bookshelf');
+
+module.exports = {
+  async getAll() {
+    const result = await knex.from('certification-cpf-countries').orderBy('commonName', 'asc');
```

Comment:
Si j'en crois la définition du modèle CertificationCpfCountry tel qu'il est aujourd'hui
```suggestion
    const results = await knex
    .select(['code', 'originalName', 'commonName']
    .from('certification-cpf-countries')
    .orderBy('commonName', 'asc');
```